CHANGELOG.md

```diff
@@ -1,3 +1,10 @@
+# Version 0.5.1
```

Comment:
Awesome :+1:. Could you also update the version in <https://github.com/phac-nml/staramr/blob/master/staramr/__init__.py>?

Reply:
Fixed in commit 22b711b